### PR TITLE
Add `SANDBOX_WORKSPACE_DIR` into config

### PR DESCRIPTION
**Pull request by xingyaoww**

### CODE REVIEW

File: opendevin/schema/config.py [hunk 1]
```diff
@@ -7,6 +7,7 @@ class ConfigType(str, Enum):
     WORKSPACE_BASE = 'WORKSPACE_BASE'
     WORKSPACE_MOUNT_PATH = 'WORKSPACE_MOUNT_PATH'
     WORKSPACE_MOUNT_REWRITE = 'WORKSPACE_MOUNT_REWRITE'
+    WORKSPACE_MOUNT_PATH_IN_SANDBOX = 'WORKSPACE_MOUNT_PATH_IN_SANDBOX'
```
Comment (li-boxuan) [inline opendevin/schema/config.py:10]: How is this config different from `WORKSPACE_MOUNT_PATH`?

Comment (xingyaoww) [reply opendevin/schema/config.py:10]: `WORKSPACE_MOUNT_PATH` is the directory of workspace folder on the host machine, and `WORKSPACE_MOUNT_PATH_IN_SANDBOX` is the folder path inside the docker sandbox.

something like `docker run -v $WORKSPACE_MOUNT_PATH:$WORKSPACE_MOUNT_PATH_IN_SANDBOX ...`

Comment (xingyaoww) [reply opendevin/schema/config.py:10]: This one!

https://github.com/OpenDevin/OpenDevin/blob/d692a72bf3809df35d802041211fcd81d56b1dc6/opendevin/sandbox/docker/ssh_box.py#L307

**Comment by li-boxuan** [comment]
You probably need to modify the comment in

https://github.com/OpenDevin/OpenDevin/blob/d692a72bf3809df35d802041211fcd81d56b1dc6/agenthub/monologue_agent/agent.py#L128

and

the prompt in

https://github.com/OpenDevin/OpenDevin/blob/d692a72bf3809df35d802041211fcd81d56b1dc6/agenthub/monologue_agent/utils/prompts.py#L61

**Comment by xingyaoww** [comment]
@li-boxuan Thanks a lot for the reminder! Just modified those!

**Review by li-boxuan — APPROVED** [review]
LGTM 👍

**Review by rbren — APPROVED** [review]

**Comment by rbren** [comment]
Thanks!

**Pull request merged**